experiment/evaluator.py

```diff
@@ -63,6 +64,8 @@ def __post_init__(self, *args, **kwargs):  # pylint: disable=unused-argument
       self.is_semantic_error = self.is_driver_fuzz_err
     if self.driver_fuzz_err:
       self.semantic_error = self.driver_fuzz_err
+    if not self.textcov_updated:
+      self.textcov_updated = textcov.Textcov()
```

Comment:
Can we use https://docs.python.org/3/library/dataclasses.html#dataclasses.field instead to initialise this?